Workiva/pdfjs_dart
# Add version safe lookup of pdfjsContext to work across pdf.js versions

Turns out, the way _I_ deployed graph_app for QA'ing this PR (https://github.com/Workiva/pdfjs_dart/pull/35) pulled in w_viewer, but not these pdfjs_dart changes. This resulted in us essentially verifying that the ETE/graph markup viewer/standard pdf viewer work with existing code. 

Thus when we went to consume the new pdfjs_dart branch, and do a cursory check on it, [we found issues](https://wf-skynet-hrd.appspot.com/apps/test/smithy/3008093/1/graph_app/sox-functional-tests#summary)! 

Basically, looking back at the changes between v1.10.90+4, and the ported over 2.5.207, [there is a different string](https://github.com/Workiva/pdfjs_dart/pull/34/files?file-filters%5B%5D=.dart#diff-37e810e81493e61ea4411c448605a974e2b9009e500bd75c131914a845cbf90bR32) passed into the context when fetching the pdfjsContext from the js files. This resulted in null reference exceptions when loading pdf.js 1.10.90+4 with the new v2.5.207 interop code.

This PR aims to 'version' safe this variable, but trying to fetch whichever context it can.

### QA instructions

- QA with https://github.com/Workiva/graph_app/pull/21916 deployed (I can do this on graphdev1). Default will be testing this interop with the 1.10.90.4 version of the pdf.js static files.
- Verify in js console the build commit versions: https://github.com/Workiva/pdfjs_dart/pull/46#issuecomment-896023395
- Ensure graph_app sox functional tests succeeded: https://wf-skynet-hrd.appspot.com/apps/test/smithy/3010656/1/graph_app/sox-functional-tests#summary
- Thoroughly smoke test ETE/graph markup viewer/standard pdf viewer paying attention to:
    - In ETE: Outline in LHP for tabs/sections of xlsx/pdfs
    - Pages load on demand when scrolling & when jumping to page
    - No unexpected exceptions surrounding pdf.js
- For extra credit:
    - You could run with a url param like this: `graph-app-pdfjs-static-file-path=https://cdn-prod.wdesk.com/pdfjs_cdn/0.1.0/assets/v1.10.90.4` and update the last version number to either v2.5.207 or v2.7.570 to observe if any noticeable difference occur across the IR viewers.
